compiler/rustc_target/src/spec/mod.rs

```diff
@@ -3503,7 +3502,15 @@ impl Target {
                     return load_file(&p);
                 }
 
-                Err(format!("Could not find specification for target {target_tuple:?}"))
+                // Leave in a specialized error message for the removed target.
+                // FIXME: If you see this and it's been a few months after this has been released,
+                // you can probably remove it.
```

Comment:
Suggestion: can we open a cleanup issue for this?

Also I'd say instead of "few months", 1 release cycle?

Reply:
this doesn't hurt much, so I'd go for a few months over 1 release cycle. it shouldn't be here forever, but I'd prefer to err on the side of longer instead of shorter.

Reply:
Seems fine.